cmd/juju/commands/ssh_common.go

```diff
@@ -31,12 +31,12 @@ import (
 // SSHCommon implements functionality shared by sshCommand, SCPCommand
 // and DebugHooksCommand.
 type SSHCommon struct {
-	modelcmd.ModelCommandBase
```

Comment:
Removing ModelCommandBase from here seems ti make things messy.
Remove IAASOnlyCommand for sure.
Maybe we should have sshContainer and sshMachine; each of those embeds SSHCommon where SSHCommon really is just the common things without having the IAAS SSH stuff still on it.

Reply:
`ModelCommandBase` wasn't removed but it was just moved from here to `sshCommand` and `scpCommand` that the places embed `SSHCommon `.
Yeah, you are right, just renamed `SSHCommon` to `SSHMachine` because it's just common for machine(IaaS) but it's not really a common thing for ssh.